src/components/create/PartBox/PartBox.jsx

```diff
@@ -79,6 +90,7 @@ const PartBox = () => {
         options={stackOptions}
         setSelectedValues={setStacks}
       />
+      <Button onClick={() => handleDelete(id)}>삭제</Button>
```

Comment:
props로 `handleDelete`를 받을 때 처음부터 `id`에 해당하는 리소스를 삭제하는 함수로 받을 수는 없나요?

```suggestion
      <Button onClick={handleDelete}>삭제</Button>
```